packages/components/src/heading/hook.ts

```diff
@@ -25,7 +25,7 @@ export interface HeadingProps extends Omit< TextProps, 'size' > {
 	/**
 	 * `Heading` will typically render the sizes `1`, `2`, `3`, `4`, `5`, or `6`, which map to `h1`-`h6`.
 	 *
-	 * @default 3
+	 * @default 2
```

Comment:
In the `useHeading` hook (defined in this same file), the default value of `level` is `2`